### DISCUSSION TITLE
Mod Compatibility

### DISCUSSION THREAD

**Discussion by KazTheMercenary** [CombatExtended-Continued/CombatExtended#1002] [General]
Hey guys,
I'm commissioning an animal armor artwork mod to add support for some of my favorite VE and Alpha Animals.  I'd love to make sure that it is both Vanilla and CE compatible.  MOST of the XML, best I can understand, is copy/paste stuff from other animal armor templates... it just needs to be balanced.

.... but I have no idea how to make sure it remains CE compatible.  And I don't want to just assume you guys will path compatibility.

SO!  How can I commission this mod, make sure it gets CE compatible, and keep it updated as painlessly as possible?

**Comment by N7Huntsman** [comment]
Generally, that's going to depend on how exactly the animal armor works, and if it varies between different animals. Essentially though, it should be a fairly simple xml patch. You can check out the existing integrated patch for stuff like Animal Armor to see how we patched that.
